.github/workflows/selenide-ci.yml

```diff
@@ -0,0 +1,164 @@
+name: Selenide Automation Tests
+
+env:
+  JAVA_VERSION: '21'
+  JAVA_DISTRIBUTION: 'temurin'
+  SELENIUM_HUB_PORT: 4444
+  SELENIUM_VERSION: '4.13.0'
+
+on:
+  push:
+    branches: [ main, master, develop, update-framework-2 ]
+  pull_request:
+    branches: [ main, master ]
+  workflow_dispatch:
+    inputs:
+      browser:
+        description: 'Browser to run tests on'
+        required: true
+        default: 'chrome'
+        type: choice
+        options:
+        - chrome
+        - edge
+      project:
+        description: 'Project to test'
+        required: true
+        default: 'VjAir'
+        type: choice
+        options:
+        - VjAir
+        - Agoda
+      headless:
+        description: 'Run in headless mode'
+        required: true
+        default: true
+        type: boolean
+      use_grid:
+        description: 'Use Selenium Grid'
+        required: false
+        default: true
+        type: boolean
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Capture start time
+        id: get_start_time
+        run: echo "start_time=$(date -u '+%Y-%m-%dT%H:%M:%SZ')" >> "$GITHUB_OUTPUT"
+
+      - name: Checkout source code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          java-version: ${{ env.JAVA_VERSION }}
+          cache: maven
+
+      - name: Setup Selenium Grid
+        if: ${{ github.event.inputs.use_grid != false }}
+        run: |
+          docker-compose up -d
```

Comment:
Hmm. I see your docker-compose file is located at resources/grid folder but didnt see you navigate to that folder before calling this command